plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -485,10 +485,21 @@ window.wpNavMenuClassChange = function ( page, url ) {
 		url === '/'
 			? 'admin.php?page=wc-admin'
 			: 'admin.php?page=wc-admin&path=' + encodeURIComponent( url );
-	const currentItemsSelector =
+	let currentItemsSelector =
 		url === '/'
 			? `li > a[href$="${ pageUrl }"], li > a[href*="${ pageUrl }?"]`
 			: `li > a[href*="${ pageUrl }"]`;
+
+	const parentPath = page.navArgs?.parentPath;
```

Comment:
`parentPath` feels a bit odd here for the use case. Could we use `navigationLinkPath` or something like that? Something that would more accurately depict what it is doing.

My only other thought would be, is to provide a callback `getNavigationLinkSelector` or something like that. I am fine with either or.

Reply:
What about `highlightedNavItemPath`? `parentNavItemPath`? `parentMenuItemPath`?

The word `parent` makes sense to my brain for this use-case.

Reply:
Right, the reason I used 'parent' is to be consistent with what's mentioned in the docs and because 'parent' is used in PHP ([ref](https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page:~:text=parent%20%E2%80%93%20Optional%20%E2%80%94%C2%A0This%20denotes%20the%20page%20as%20a%20child%20of%20parent%20(using%20the%20parent%E2%80%99s%20ID)%20and%20is%20used%20for%20generating%20breadcrumbs.)).

Too bad that there is no way to access the PHP registered pages and just used the parent and child relation from that data since in PHP, `parent`s are set. If that is possible, that would be an elegant solution.

Anyway, back to reality, naming is hard. Whatever we choose, as long as documented well in https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page, it's OK IMO.

Talking about docs, just noting here that a change to https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page will be needed after this change is released. I'll create a follow-up issue after this PR is merged.

Reply:
Too bad that there is no way to access the PHP registered pages and just used the parent and child relation from that data since in PHP, `parent`s are set. If that is possible, that would be an elegant solution.

Anyway, back to reality, naming is hard. Whatever we choose, as long as documented well in https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page, it's OK IMO.

Talking about docs, just noting here that a change to https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page will be needed after this change is released. I'll create a follow-up issue after this PR is merged.

Reply:
> Right, the reason I used 'parent' is to be consistent with what's mentioned in the docs and because 'parent' is used in PHP ([ref](https://developer.woocommerce.com/extension-developer-guide/working-with-woocommerce-admin-pages/#registering-a-react-powered-page:~:text=parent%20%E2%80%93%20Optional%20%E2%80%94%C2%A0This%20denotes%20the%20page%20as%20a%20child%20of%20parent%20(using%20the%20parent%E2%80%99s%20ID)%20and%20is%20used%20for%20generating%20breadcrumbs.)).

Gotcha, yeah I guess that makes sense, and currently our react router seems to only support a linear list of routes, so we can't make use of React router to get the parent instead.
If we forget about it only being used for highlighting right now, I think `parentPath` makes sense, and may be something beneficial in the future.
In one way I still think that the highlighting of the parent is semi unique, and I would almost want to include an additional prop with the `parentPath` - `highlightParent` or something to that affect, but we can stick with this approach for now.